src/core/reducers/addons.js

```diff
@@ -195,7 +203,13 @@ export function createInternalAddon(
     guid: apiAddon.guid,
     has_eula: apiAddon.has_eula,
     has_privacy_policy: apiAddon.has_privacy_policy,
-    homepage: selectLocalizedContent(apiAddon.homepage, lang),
+    homepage:
+      homepage_url && homepage_outgoing
```

Comment:
I'd introduce a selector to handle this chunk of code and the few added lines above:

```js
const selectLocalizedOutgoingUrl = (url, lang) => {
  if (url && url.url && url.outgoing) {
    return {
      url: selectLocalizedContent(url.url, lang),
      outgoing: selectLocalizedContent(url.outgoing, lang),
    };
  }

  return null;
};
```

then you'd use it here like this:

```js
[...]
homepage: selectLocalizedOutgoingUrl(apiAddon.homepage, lang),
```

Reply:
https://github.com/mozilla/addons-frontend/pull/9960/commits/22475b17568b308fe64c49362ff08a61242f669a